lib/core/src/client/preview/url.ts

```diff
@@ -60,8 +60,16 @@ const deprecatedLegacyQuery = deprecate(
 Use \`id=$storyId\` instead. 
 See https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-url-structure`
 );
-export const getSelectionSpecifierFromPath: () => StoreSelectionSpecifier = () => {
+
+// Lifted from @storybook/router utils
```

Comment:
We can't import? 🤔

Reply:
No, we have no shared code between manager and preview right now (`core-events` being the sole exception, but thats just a bunch of constants). Norbert and Michael are working on something which will introduced a `core-common` package, but until then it's easier to copy/paste.